### PR TITLE
Certificate restyle according to edX

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots
  - [x] Mobile width screenshots
- [x] Testing
  - [x] Changes have been manually tested

#### What are the relevant tickets?
https://github.com/mitodl/micromasters/issues/4788

#### What's this PR do?
It redesigns micromasters certificate templates according to edX's certificates.
An [edx.org certificates](https://courses.edx.org/certificates/dd6838d94f4b41038c03be77aed91172)
Our current certificates:
[course certificates look like this](https://micromasters-rc.odl.mit.edu/certificate/course/74fa750f2d30c4cfa8bfc21052d88056) and [program certificates look like this](https://micromasters-rc.odl.mit.edu/certificate/program/0d54d09954752e507775550841644584).

**Note:**
As the goal was to get similar experience as edx.org certificates have, most of the design and css is taken from edX and then customized according to our platform.

#### How should this be manually tested?
1) Generate course and program certificates, and compare them with edx.org's ceritificates.
2) Make sure that no CSS is breaking on other pages of the platform. 

#### Screenshots (if appropriate)
**Desktop Version:**
Single Course:
image

Program:
image

**Printed Version:**
Course:
image

Program:
image

**Mobile Version:**

### CODE REVIEW

File: ui/templates/background-images.css [hunk 1]
```diff
@@ -20,21 +20,16 @@
...
-
-@media print {
-  .certificate .certificate-body.verified {
-    -webkit-print-color-adjust: exact !important;   /* Chrome, Safari */
-    color-adjust: exact !important;  /* Firefox */
-    background-image: url({% static "images/mm_watermark2.png" %}) !important;
-    background-size: cover !important;
+@media (min-width: 1200px) {
+  .certificate .accomplishment-rendering {
+    background-image: url({% static "images/micromasters_certificate_watermark.svg" %}) !important;
```
Comment (umarmughal824) [inline ui/templates/background-images.css:28]: @HamzaIbnFarooq these two lines are needed here, could you add that back?
These are required to show the background watermark in the print background.

```
-webkit-print-color-adjust: exact !important;   /* Chrome, Safari */	
color-adjust: exact !important;  /* Firefox */```

File: ui/templates/background-images.css [hunk 2]
```diff
@@ -20,21 +20,16 @@
...
   }
 }
-
-.b2b {
-  background-image: url({% static "images/b2b_section.png" %});
+@media print {
```
Comment (umarmughal824) [inline ui/templates/background-images.css:31]: @HamzaIbnFarooq these two lines are needed here, could you add that back?
These are required to show the background watermark in the print background.

```
-webkit-print-color-adjust: exact !important;   /* Chrome, Safari */	
color-adjust: exact !important;  /* Firefox */```

Comment (umarmughal824) [inline ui/templates/background-images.css:31]: @HamzaIbnFarooq these two lines are needed here, could you add that back?
These are required to show the background watermark in the print background.

```
-webkit-print-color-adjust: exact !important;   /* Chrome, Safari */	
color-adjust: exact !important;  /* Firefox */```

File: ui/templates/background-images.css [hunk 3]
```diff
@@ -20,21 +20,16 @@
...
+  .certificate .accomplishment-rendering {
+    background-image: url({% static "images/micromasters_certificate_watermark.svg" %}) !important;
+  }
 }
```
Comment (umarmughal824) [inline ui/templates/background-images.css:35]: that part got skipped while rebasing
```
@media print and (orientation: landscape) {
  body {
    display: flex;
    justify-content: center;
    align-items: center;
  }
}
```

**Comment by umarmughal824** [comment]
@pdpinch could you review the attached screenshots?

File: ui/templates/certificate.html [hunk 4]
```diff
@@ -10,120 +10,130 @@
...
-        <div class="col-md-4 certificate-date">
-          <label>Verified Certificate</label>
-          <span>Issued {{ certificate.created_on|date:"F j, Y" }}</span>
+        <div class="wrapper-accomplishment-statement">
+          <div class="accomplishment-statement">
+            <p class="accomplishment-statement-lead">
+              <span class="accomplishment-statement-detail">This is to certify that</span>
+              <strong class="accomplishment-recipient">{{ name }}</strong>
+              {% if course_title %}
+                <span class="accomplishment-summary">has successfully completed and received a passing grade in</span>
```
Comment (pdpinch) [inline ui/templates/certificate.html:71]:
```suggestion
                <span class="accomplishment-summary">successfully completed and received a passing grade in</span>
```

File: ui/templates/certificate.html [hunk 5]
```diff
@@ -10,120 +10,130 @@
...
+                <span class="accomplishment-course">
+                  <span class="accomplishment-course-name">{{ course_title }}</span>
+                </span>
+                <span class="accomplishment-statement-detail">a course offered by the {{ program_title }} MicroMasters program,
+                  an online learning initiative offered by the Massachusetts Institute of Technology in collaboration with edX.</span>
```
Comment (pdpinch) [inline ui/templates/certificate.html:76]:
```suggestion
                <span class="accomplishment-statement-detail">a course of study offered by MITx,
                  an online learning initiative of the Massachusetts Institute of Technology.</span>
```

File: ui/templates/certificate.html [hunk 6]
```diff
@@ -10,120 +10,130 @@
...
+                </a>
+              </li>
+              <li>
+                <a href="javascript:window.print();">
+                    <img src="{% static 'images/certificates/icon-print.svg' %}" alt="Print">
+                </a>
+              </li>
+            </ul>
+            <h2>Congratulations, {{ name }}!</h2>
+            <p>You have successfully completed {% if course_title %}{{ course_title }}{% else %}{{ program_title }}{% endif %}. Share your accomplishment with your friends, family and colleagues. </p>
```
Comment (pdpinch) [inline ui/templates/certificate.html:40]:
```suggestion
            <p>You worked hard to earn your certificate from MITx — share it with colleagues, friends, and family to get the word out about what you mastered in {% if course_title %}{{ course_title }}{% else %}{{ program_title }}{% endif %}.</p>
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
The styling looks great, but I noticed some text changes in the edX version which I think we should copy here. 

If you can give me an updated set of screenshots tomorrow, I think I should run this by someone from the MicroMasters team to make sure we have all the text correct.

File: ui/templates/background-images.css (hunk 2 above)
Comment (HamzaIbnFarooq) [reply ui/templates/background-images.css:31]: Thanks, I thought the background can only be printed using browser settings. Updated PR!

File: ui/templates/background-images.css (hunk 3 above)
Comment (HamzaIbnFarooq) [reply ui/templates/background-images.css:35]: Updated.

File: ui/templates/certificate.html (hunk 4 above)
Comment (HamzaIbnFarooq) [reply ui/templates/certificate.html]: Do we have to change the text for program certificates as well? I have added screenshots in the description you can have a clear comparison there.

**Comment by HamzaIbnFarooq** [comment]
@pdpinch 
Just to make it prominent, the edX logo used on edx.org certificates doesn't exactly match the trademark we have been using throughout micromasters, do we need to change it as well in certificates?
Micromaster's edX logo:


edx.org's edX logo:

**Review by umarmughal824 — APPROVED** [review]
LGTM,
Just a minor concerns

1. please move all the certificates images to `static/images/certificate`
2. just add a slight margin on print for the logo to move it to the left side so that it could reach within the margin frame

Screenshot 2021-02-26 at 15 51 06

File: ui/templates/certificate.html (hunk 4 above)
Comment (pdpinch) [reply ui/templates/certificate.html]: Yes, we should change them both to match edX.

**Comment by pdpinch** [comment]
Ok, I've gotten confirmation that we want this to look just like the edX version, so I think is ready to merge.

**Review by pdpinch — APPROVED** [review]

**Pull request merged**